oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -267,6 +265,37 @@ discussion category
 <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_ or in the
 ``#core-contributors`` room on Slack.
 
+Removing Core Contributors
+--------------------------
+
+Occasionally, it may become necessary to remove an individual from the Core
+Contributor program. Some reasons this may happen include:
+
+* A CC leaves the project or changes employment and does not wish to continue
+  participating
+* A CC does not have the time or interest to continue in the role
+* A CC is not fulfilling the responsibilities of the role
+* A CC is not upholding the standards of the community
+
+Trigger events for removing a CC would be:
+
+* A CC informs the Program Administrators that they can no longer continue in
+  the role
+* A CC has been inactive and unreachable for more than 6 months
+* A CC hasn't met project quality standards consistently despite feedback
+* A CC has consistently shown poor judgement, such as merging PRs without
+  due consideration or addressing outstanding concerns
+* A CC has violated the project `Code of Conduct`_
```

Comment:
@sarina I'm coming a bit late for the review here it seems -- but what about the cases where core contributors are somewhat active, but don't fulfill the commitments made? With just a vague "activity" requirement, don't we run the risk of ending up with many core contributors who don't do much?

Reply:
We discussed this at tCRIL and also I did some reading on healthy communities. Having a core contributor who contributes a few patches a year, or only does review, or reviews translations only once per month or quarter... well, that work _is_ valuable. And just because they may be very busy right now does not mean they will be in the future.

We do say below,
> Program administrators will be responsible for determining when a CC is inactive/unreachable, by doing a check at least once per quarter.

I think as part of doing the check we will begin to understand if this concern is warranted or not. As is my prerogative, I tend to not want to overly-solve for situations until we encounter the situation - with context - and figure out where to go from there.

Reply:
@sarina OK, we can see how that goes. But imho we are in a special case of a community, where a lot of the parties deciding on where/how much work is contributed are companies -- not the core contributors themselves. And the goal with this is to help the core contributors getting that time allocated by their bosses, by providing a concrete incentive for the company itself (the claim of having X core contributors). Some companies will play fair regardless, but imho for others if they can get the outcome without having to prioritize it over other company priorities, they won't - and we'll have core contributors saying "we have been busy at our company, so I haven't been able to spend on this".

Reply:
Submitted at https://github.com/openedx/open-edx-proposals/pull/304